server/src/main/scala/de/frosner/broccoli/signal/SignalModule.scala

```diff
@@ -0,0 +1,14 @@
+package de.frosner.broccoli.signal
+
+import javax.inject.Singleton
+
+import com.google.inject.{AbstractModule, Provides}
+import net.codingwell.scalaguice.ScalaModule
+
+class SignalModule extends AbstractModule with ScalaModule {
+  override def configure(): Unit = {}
+
+  @Provides
+  @Singleton
+  def provideSignalManager(): UnixSignalManager = new UnixSignalManager()
```

Comment:
Please use a simple `.bind` expression in `configure`; we don't need a provider method here.